diff --git a/bin/bout-config.in b/bin/bout-config.in
index 15da5abf..5cf0c5f7 100755
--- a/bin/bout-config.in
+++ b/bin/bout-config.in
@@ -19,8 +19,9 @@ cc="@MPICXX@"
 cxx="@MPICXX@"
 ld="@MPICXX@"
 checks="@CHECK_LEVEL@"
-cflags="@CONFIG_CFLAGS@"
-libs="@CONFIG_LDFLAGS@"
+
+cflags="$CXXFLAGS -I$BOUT_INCLUDE_PATH $CXXINCLUDE @CXXFLAGS@ @OPENMP_CXXFLAGS@ @CXX11_FLAGS@ @COVERAGE_FLAGS@ @EXTRA_INCS@"
+libs="-lm -L$BOUT_LIB_PATH -lbout++ @LDFLAGS@ @EXTRA_LIBS@ @OPENMP_CXXFLAGS@"
 
 version="@BOUT_VERSION@"
 git="@GIT_REVISION@"
diff --git a/configure.ac b/configure.ac
index 3853a467..65544b65 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,4 +1,4 @@
-# Copyright 2010 B D Dudson, S Farley
+***# Copyright 2010 B D Dudson, S Farley
 #
 # Contact Ben Dudson, bd512@york.ac.uk
 #
diff --git a/tests/integrated/test-fci-slab/runtest b/tests/integrated/test-fci-slab/runtest
index 689d283f..e6d819d2 100755
--- a/tests/integrated/test-fci-slab/runtest
+++ b/tests/integrated/test-fci-slab/runtest
@@ -122,34 +122,37 @@ for var,mark,label in zip(varlist, markers, labels):
         print("............ FAIL")
         success = False
 
-try:
-    # Plot using matplotlib if available
-    import matplotlib.pyplot as plt
-    
-    plt.figure()
+if False:
+    try:
+        # Plot using matplotlib if available
+        import matplotlib.pyplot as plt
 
-    for var,mark,label in zip(varlist, markers, labels):
-        plt.plot(dx, error_2[var], '-'+mark, label=label)
-        plt.plot(dx, error_inf[var], '--'+mark)
-    
-    plt.legend(loc="upper left")
-    plt.grid()
+        plt.figure()
+
+        for var,mark,label in zip(varlist, markers, labels):
+            plt.plot(dx, error_2[var], '-'+mark, label=label)
+            plt.plot(dx, error_inf[var], '--'+mark)
 
-    plt.yscale('log')
-    plt.xscale('log')
+        plt.legend(loc="upper left")
+        plt.grid()
 
-    plt.xlabel(r'Mesh spacing $\delta x$')
-    plt.ylabel("Error norm")
+        plt.yscale('log')
+        plt.xscale('log')
 
-    plt.savefig("fci-norm.pdf")
+        plt.xlabel(r'Mesh spacing $\delta x$')
+        plt.ylabel("Error norm")
 
-    print("Plot saved to fci-norm.pdf")
+        plt.savefig("fci-norm.pdf")
 
-    if showPlot:
-        plt.show()
-    plt.close()
-except ImportError:
-    print("No matplotlib")
+        print("Plot saved to fci-norm.pdf")
+
+        if showPlot:
+            plt.show()
+        plt.close()
+    except ImportError:
+        print("No matplotlib")
+else:
+    print("Pltting disabled")
 
 if success:
     exit(0)
diff --git a/tools/pylib/_boutcore_build/Makefile b/tools/pylib/_boutcore_build/Makefile
index 95760c5d..13631e1f 100644
--- a/tools/pylib/_boutcore_build/Makefile
+++ b/tools/pylib/_boutcore_build/Makefile
@@ -11,7 +11,7 @@ helper.cxx:helper.cxx.in
 TOGEN=setup.py boutcore.pyx resolve_enum.pxd helper.cxx helper.h
 $(TOGEN): Makefile
 	@echo generating $@
-	@PATH=../../../bin:$(PATH) bash $@.in > $@
+	@PATH=../../../bin:$$PATH bash $@.in > $@
 
 boutcore.so: boutcore.pyx setup.py helper.cxx helper.h $(TOGEN)
 	@$(PY) setup.py build_ext --inplace
diff --git a/tools/pylib/_boutcore_build/boutcore.pyx.in b/tools/pylib/_boutcore_build/boutcore.pyx.in
index d264d980..046c10d9 100755
--- a/tools/pylib/_boutcore_build/boutcore.pyx.in
+++ b/tools/pylib/_boutcore_build/boutcore.pyx.in
@@ -455,7 +455,7 @@ def bracket(Field3D a, Field3D b, method="BRACKET_STD", outloc="CELL_DEFAULT"):
     return f3dFromObj(c.bracket(a.cobj[0],b.cobj[0],method_,outloc_))
 
 EOF
-for fun in sqrt exp sin cos log
+for fun in sqrt exp sin cos log abs
 do
     echo "def $fun(Field3D a):
     return f3dFromObj(c.$fun(a.cobj[0]))
@@ -470,8 +470,8 @@ def pow(Field3D a, exponent):
 def min(Field3D a):
     return c.min(a.cobj[0])
 
-def min(Field3D a):
-    return c.min(a.cobj[0])
+def max(Field3D a):
+    return c.max(a.cobj[0])
 
 def Delp2(Field3D a, zsmooth=-1):
     return f3dFromObj(c.Delp2(a.cobj[0],float(zsmooth)))
diff --git a/tools/pylib/_boutcore_build/boutcpp.pxd b/tools/pylib/_boutcore_build/boutcpp.pxd
index b1054ff1..99ff7ed5 100644
--- a/tools/pylib/_boutcore_build/boutcpp.pxd
+++ b/tools/pylib/_boutcore_build/boutcpp.pxd
@@ -20,6 +20,7 @@ cdef extern from "field3d.hxx":
     Field3D log(Field3D)
     Field3D sin(Field3D)
     Field3D cos(Field3D)
+    Field3D abs(Field3D)
     double max(Field3D)
     double min(Field3D)
     Field3D pow(Field3D,double)
