diff -Naur orginal/BOUT-dev-4.0.0/make.config.in ./BOUT-dev-4.0.0/make.config.in
--- a/make.config.in	2017-02-03 16:41:27.000000000 +0000
+++ b/make.config.in	2017-05-02 17:32:14.653742067 +0100
@@ -30,7 +30,7 @@
 AR			= ar
 ARFLAGS                 = @ARFLAGS@
 RANLIB			= @RANLIB@
-MKDIR			= @MKDIR@
+MKDIR			= @MKDIR_P@
 
 ####################################################################
 # Do not need to alter anything below
@@ -98,11 +100,15 @@
 prefix = @prefix@
 exec_prefix = @exec_prefix@
 install: all
-	@$(MKDIR) $(DESTDIR)${prefix}
-	@$(MKDIR) $(DESTDIR)${prefix}/include
-	@$(MKDIR) $(DESTDIR)${prefix}/lib
+	@$(MKDIR) $(DESTDIR)@includedir@
+	@$(MKDIR) $(DESTDIR)@includedir@/bout
+	@$(MKDIR) $(DESTDIR)@includedir@/bout/sys
+	@$(MKDIR) $(DESTDIR)@libdir@
 	@echo "Installing header files into $(DESTDIR)@includedir@"
 	@cp include/*.hxx $(DESTDIR)@includedir@
+	@cp include/bout/*.hxx $(DESTDIR)@includedir@/bout
+	@cp include/bout/sys/*.hxx $(DESTDIR)@includedir@/bout/sys
+	@cp make.config $(DESTDIR)@includedir@/bout/make.config
 	@echo "Installing library files into $(DESTDIR)@libdir@"
 	@cp lib/*.a $(DESTDIR)@libdir@
