--- a/make.config.in	2017-05-02 17:31:36.531617486 +0100
+++ b/make.config.in	2017-05-02 17:32:14.653742067 +0100
@@ -58,7 +58,9 @@
 # Files that are needed by configure and should be dependencies for 'all'
 
 OBJ						= $(SOURCEC:%.cxx=%.o)
+ifndef RELEASE
 LIB						= $(BOUT_TOP)/lib/libbout++.a
+endif
 
 BOUT_INCLUDE	= -I$(BOUT_TOP)/include $(CXXINCLUDE) $(EXTRA_INCS)
 BOUT_LIBS			= -lm -L$(BOUT_TOP)/lib -lbout++ $(EXTRA_LIBS)
