--- a/configure.ac	2017-02-03 16:41:27.000000000 +0000
+++ b/configure.ac	2017-04-28 02:02:26.000435692 +0100
@@ -594,12 +594,19 @@
         ))
   fi
   
+  PETSC_INC=$PETSC_DIR/include
+  if test "$with_petsc" == "fedora"
+  then
+      PETSC_DIR=/usr
+      PETSC_INC=/usr/include/petsc
+  fi
+
   PETSC_VERSION_MAJOR="0"
-  PETSC_VERSION_MAJOR=`grep 'define PETSC_VERSION_MAJOR' $PETSC_DIR/include/petscversion.h | awk '{print \$3;}'`
+  PETSC_VERSION_MAJOR=`grep 'define PETSC_VERSION_MAJOR' $PETSC_INC/petscversion.h | awk '{print \$3;}'`
   PETSC_VERSION_MINOR="0"
-  PETSC_VERSION_MINOR=`grep 'define PETSC_VERSION_MINOR' $PETSC_DIR/include/petscversion.h | awk '{print \$3;}'`
+  PETSC_VERSION_MINOR=`grep 'define PETSC_VERSION_MINOR' $PETSC_INC/petscversion.h | awk '{print \$3;}'`
   PETSC_VERSION_RELEASE="0"
-  PETSC_VERSION_RELEASE=`grep 'define PETSC_VERSION_RELEASE' $PETSC_DIR/include/petscversion.h | awk '{print \$3;}'`
+  PETSC_VERSION_RELEASE=`grep 'define PETSC_VERSION_RELEASE' $PETSC_INC/petscversion.h | awk '{print \$3;}'`
 
   if test $PETSC_VERSION_MAJOR != "3"
   then
@@ -656,8 +663,14 @@
 MFILE
 
   AC_MSG_CHECKING([PETSC has C++ support])
-  PETSC_HAS_CXX=`make -f petscmake$$ echo_cxx`
-
+  if test "$with_petsc" == "fedora"
+  then
+      PETSC_HAS_CXX="yes"
+      AC_MSG_WARN([Ignoring test ...])
+  else
+      echo $with_petsc
+      PETSC_HAS_CXX=`make -f petscmake$$ echo_cxx`
+  fi
   AS_IF([test -z "$PETSC_HAS_CXX"],
   [
     AC_MSG_RESULT([no])
@@ -680,7 +693,14 @@
   rm petscmake$$
 
   # Set the line to be included in the make.conf file
-  PETSC="include ${PETSC_CONFDIR}/variables"
+  if test "x$with_petsc" != "xfedora"
+  then
+      PETSC="include ${PETSC_CONFDIR}/variables"
+  else
+      PETSC=""
+      EXTRA_INCS+=" -I\$(MPI_INCLUDE)/petsc -I\$(MPI_INCLUDE)/petsc/petsc/finclude"
+      EXTRA_LIBS+=" -lpetsc"
+  fi
 else
   PETSC=
 fi
